cache/cache_test.cc

```diff
@@ -26,16 +22,32 @@
 namespace ROCKSDB_NAMESPACE {
 
 // Conversions between numeric keys/values and the types expected by Cache.
```

Comment:
Pls try to put these functions into an anonymous namespace, so that they are translation unit local ones.